features/facebookImport/src/model/importers/language-and-locale-importer.js

```diff
@@ -4,6 +4,7 @@ import { readJSONDataArray } from "./utils/importer-util.js";
 export const LANGUAGE_AND_LOCALE_FILE_PATH =
     "preferences/language_and_locale.json";
 export const LANGUAGE_AND_LOCALE_DATA_KEY = "language_and_locale_v2";
+export const LANGUAGE_AND_LOCALE_STORAGE_KEY = "languageAndLocale";
```

Comment:
I am not sure we should separate the storage key for this. Except in cases like the "Old" classes (which could be eliminated anyway, or could be hacked to generate the key) the key can be computed from the class name, or the file name.

Reply:
Oh, yeah that's what I suggested _before_ reading your comment :D But IMHO we should do it in a followup, since currently these keys are being used, it's best to keep them around for now while we're changing so much, I think.